package.json

```diff
@@ -21,16 +21,17 @@
     "test:e2e": "jest --config ./test/jest-e2e.json --runInBand",
     "test:e2e:cov": "jest --coverage --config ./test/jest-e2e.json --runInBand",
     "test:all": "jest --config ./test/jest-all.json --runInBand",
-    "test:all:cov": "jest --coverage --config ./test/jest-all.json --runInBand",
-    "postinstall": "husky install"
```

Comment:
We should mention that git hooks are no longer installed automatically after project installation.

My suggestion is to write something like [this section](https://github.com/safe-global/safe-gelato-relay-service/blob/43f49bc377dbf658033bb2eb984e04f431e7b4cb/README.md?plain=1#L86-L95)

Reply:
Added in [700b4a2](https://github.com/safe-global/safe-client-gateway-nest/pull/522/commits/700b4a242a29ead048e1ccee36cd4207ccc8ec9a), thanks!!